tests/json_dump.gtest.cpp

```diff
@@ -35,7 +35,7 @@ static void check_full_json_dump(const std::string& string_dump) {
 
 	for (rapidjson::Value::MemberIterator iter = dump.MemberBegin(); iter != dump.MemberEnd(); ++iter) {
 		ASSERT_TRUE(iter->value.IsObject());
-		ASSERT_TRUE(iter->value.FindMember("type") != NULL);
+		ASSERT_TRUE(iter->value.FindMember("type") != iter->value.MemberEnd());
```

Comment:
You could use `HasMember()` instead.

Reply:
fixed https://github.com/yandex/handystats/pull/16/commits/53fa17aa4533bee7afd4a868f0984c9057e2bb3b